### ISSUE TITLE
COE Punch List - Part II

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#38986]
## Background
COE is getting its last punchlist, before last check and final check

## Consideration
- Punch List from Jim: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/ebenefits/features/apply-for-home-loan-COE/research-design/coe-design-punchlist.md
- Form intro action link probably needs to be done first, as Kelli will ask (in another ticket)

## Tasks
- [ ] Implement what can be done by the FE

## Acceptance Criteria
- [ ] Punch list items have been addressed by FE
- [ ] A PR has been requested for merging
